### DISCUSSION TITLE
Batched Faster Whisper: The aguments of BatchedInferencePipeline

### DISCUSSION THREAD

**Discussion by toanhn** [SYSTRAN/faster-whisper#1033] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/de42e2b5-c6df-428f-8aa7-5d74e96bf228)
What do these arguments mean?
- vad_onset
- vad_offset

Thanks.

**Comment by MahmoudAshraf97** [comment, marked as answer]
`vad_onset`: the threshold where if the probability is higher than this, then it's classified as speech
`vad_offset`: if the previous segment is considered speech, all consecutive segments are still speech even if they are less than `vad_onset`, it is only considered silence if it's less than `vad_offset`

**Reply by toanhn** [reply]
Thanks, friend.

**Reply by MahmoudAshraf97** [reply]
My Pleasure, can you please mark this as an answer and close the discussion as completed?
